mint.json

```diff
@@ -127,118 +126,139 @@
     },
     {
       "group": "Architecture",
-      "pages": ["architecture/introduction", "architecture/diagrams"]
-    },
-    {
-      "group": "Build Workflows",
       "pages": [
-        "workflow/introduction",
-        "workflow/content",
-        "workflow/digest",
-        "workflow/delay"
+        "architecture/overview",
+        "architecture/diagrams"
       ]
     },
-         
     {
-      "group": "Channels",
+      "group": "Build Workflows",
       "pages": [
+        "workflow/overview",
         {
-          "group": "In-App",
-          "icon": "bell",
-          "pages": ["integrations/providers/in-app/overview"]
+          "group": "How-To",
+          "pages": [
+            "workflow/how-to/build-a-workflow"
+          ]
```

Comment:
Let's remove this group and just call it Building workflows

Reply:
Done.